real_estate/__init__.py

```diff
@@ -0,0 +1,3 @@
+# __init__
+from . import models
+#from . import wizard
```

Comment:
```suggestion
from . import models
```
Puedes eliminar las líneas innecesarias